packages/common/src/utils/signature_utils.ts

```diff
@@ -0,0 +1,71 @@
+import type { Cacao } from 'ceramic-cacao'
+import type { DID } from 'dids'
+import { CommitData } from '../index.js'
+import { StreamID } from '@ceramicnetwork/streamid'
```

Comment:
```suggestion
import type { CommitData } from '../index.js'
import type { StreamID } from '@ceramicnetwork/streamid'
```
I believe we only need type here?